src/test/java/com/puppycrawl/tools/checkstyle/PackageObjectFactoryTest.java

```diff
@@ -46,6 +46,11 @@ public void testCtorException() {
         new PackageObjectFactory(new HashSet<>(), null);
     }
 
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
please do not use that property of annotation, you do not check exact exception.
You are ok with any IllegalArgumentException , it is not correct. Please use catch and assert by message.
One day it , automation of this will be finished - https://github.com/sevntu-checkstyle/sevntu.checkstyle/issues/108